nav2_mission_executor/src/mission_executor.cpp

```diff
@@ -14,41 +14,123 @@
 
 #include "nav2_mission_executor/mission_executor.hpp"
 
+#include <string>
 #include <memory>
-#include "nav2_mission_executor/execute_mission_behavior_tree.hpp"
 
-using nav2_tasks::TaskStatus;
+#include "nav2_mission_executor/execute_mission_behavior_tree.hpp"
 
 namespace nav2_mission_executor
 {
 
 MissionExecutor::MissionExecutor()
-: Node("MissionExecutor")
+: nav2_lifecycle::LifecycleNode("mission_executor", "", true)
+{
+  RCLCPP_INFO(get_logger(), "Creating");
+}
+
+MissionExecutor::~MissionExecutor()
+{
+  RCLCPP_INFO(get_logger(), "Destroying");
+}
+
+nav2_lifecycle::CallbackReturn
+MissionExecutor::on_configure(const rclcpp_lifecycle::State &)
```

Comment:
Same as my previous comments, we've been using `\*...*\` for unused parameters.

Reply:
Fixed